jenkins/scripts/pulse_in_pipeline_scanning/utils/common.py

```
@@ -31,7 +36,10 @@ def is_permissive(licenses: list, license_check_token: str) -> dict:
         if response:
             resp = response.json()
             if resp["success"]:
-                return {lic: result["isPermissive"] for lic, result in zip(licenses, resp["data"])}
+                return {
+                    lic: license_needs_review(lic, result)
+                    for lic, result in zip(licenses, resp["data"])
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)